apps/table.html

```diff
@@ -415,13 +444,20 @@ <h5 class="modal-title" id="slideNameChangeModalLabel">Slide name change confirm
 
 	function initialize() {
 		slideDeleteRequests = [];
+		userCreateRequests = [];
 		const params = getUrlVars();
 		const store = new Store('../data/');
 		store.findRequest()
 			.then(function (requests) {
 				// console.log(requests);
 				if (requests) {
```

Comment:
Missed this before:
```suggestion
				if (requests && ! requests.error) {
```
Or maybe
```suggestion
				if (requests && requests.forEach) {
```
Or the below. Or something else :) just to avoid failing if you get a 401 (or other error) on fetching requests.